config.yaml

```diff
@@ -38,15 +38,18 @@ params:
   mash:
     sketch_size: 1000
   kraken:
-    kraken_db: test_data/kraken_test_db
-    bracken_db: test_data/kraken_test_db/75mers_distr.txt
-    # kraken_db: /home/qiz173/Works/WoL/profiling/ref107/kraken
-    # bracken_db: /home/qiz173/Works/WoL/profiling/ref107/kraken/75mers_distr.txt
+    db: test_data/kraken_test_db
+    kmers: test_data/kraken_test_db/75mers_distr.txt
+    # db: /home/qiz173/Works/WoL/profiling/ref107/kraken
```

Comment:
Please delete these commented lines

Reply:
Oh @tanaes I don't think so. I still need to test the programs locally. Deleting them will disable me from doing so.

Reply:
The better option here is to keep a local copy with those values that isn't merged to the central repo. If we merge these commented lines it will get pushed to all devs' branches.